core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -9,57 +9,70 @@ import "../../oracle/interfaces/StoreInterface.sol";
     FinderInterface public finder;
 
-    /**
-     * Tracks the last block time when the fees were paid.
-     */
+    // Tracks the last block time when the fees were paid.
     uint public lastPaymentTime;
 
-    /**
-     * Tracks the cumulative fees that have been paid by the contract for use by derived contracts.
-     * The multiplier starts at 1, and is updated by computing cumulativeFeeMultiplier * (1 - effectiveFee).
-     * Put another way, the cumulativeFeeMultiplier is (1 - effectiveFee1) * (1 - effectiveFee2) ...
-     *
-     * For example:
-     *
-     * The cumulativeFeeMultiplier should start at 1.
-     * If a 1% fee is charged, the multiplier should update to .99.
-     * If another 1% fee is charged, the multiplier should be 0.99^2 (0.9801).
-     */
+    // Tracks the cumulative fees that have been paid by the contract for use by derived contracts.
+    // The multiplier starts at 1, and is updated by computing cumulativeFeeMultiplier * (1 - effectiveFee).
+    // Put another way, the cumulativeFeeMultiplier is (1 - effectiveFee1) * (1 - effectiveFee2) ...
+    // For example:
+    // The cumulativeFeeMultiplier should start at 1.
+    // If a 1% fee is charged, the multiplier should update to .99.
+    // If another 1% fee is charged, the multiplier should be 0.99^2 (0.9801).
     FixedPoint.Unsigned public cumulativeFeeMultiplier;
 
-    /**
-     * @notice modifier that calls payFees().
-     */
+    /****************************************
+     *              MODIFIERS               *
+     ****************************************/
+
+    // modifier that calls payFees().
     modifier fees {
         payFees();
         _;
     }
 
-    constructor(address collateralAddress, address finderAddress, bool _isTest) public Testable(_isTest) {
+    /**
+     * @notice constructs the FeePayer contract. Called by parent contracts.
+     * @param collateralAddress ERC20 token that is used as the underlying collateral for the synthetic.
+     * @param finderAddress UMA protocol finder used to discover other protocol contracts.
+     * @param isTest whether this contract is being constructed for the purpose of running tests.
+     */
+    constructor(address collateralAddress, address finderAddress, bool isTest) public Testable(isTest) {
         collateralCurrency = IERC20(collateralAddress);
         finder = FinderInterface(finderAddress);
         lastPaymentTime = getCurrentTime();
         cumulativeFeeMultiplier = FixedPoint.fromUnscaledUint(1);
     }
 
+    /****************************************
+     *        FEE PAYMENT FUNCTIONS         *
+     ****************************************/
+
     /**
-     * @notice Pays UMA DVM regular fees to the Store contract. These must be paid periodically for the life of the contract.
+     * @notice Pays UMA DVM regular fees to the Store contract.
+     * @dev These must be paid periodically for the life of the contract.
```

Comment:
nit: you might want to throw a note here about the caller getting paid if the contract has not paid regular fees in a week or more.